.ci/r_tests_windows.ps1

```diff
@@ -50,6 +50,7 @@ Invoke-WebRequest -Uri https://sourceforge.net/projects/qpdf/files/qpdf/10.3.1/q
 Copy-Item .\qpdf -Destination $env:R_LIB_PATH -Recurse
 
 initexmf --set-config-value [MPM]AutoInstall=1
+$env:MIKTEX_EXCEPTION_PATH = "$env:TEMP\miktex"
```

Comment:
This is to avoid the following new CRAN NOTE on Windows:
```
* checking for detritus in the temp directory ... NOTE
Found the following files/directories:
  'lastMiKTeXException'
```

Env variable was found in sources of MikTeX:
https://github.com/MiKTeX/miktex/blob/ab8ebca7c70fe8c9a1392dfb2393a0a7683e14cc/Libraries/MiKTeX/Core/Exceptions.cpp#L155-L183
https://github.com/MiKTeX/miktex/blob/ab8ebca7c70fe8c9a1392dfb2393a0a7683e14cc/Libraries/MiKTeX/Core/include/miktex/Core/Environment.h#L37

We cannot remove MikTeX temp files during running `testthat` because PDF generation happens later in CRAN checks after tests have been finished.
Ideally, this path should be controlled via `TempDir` config value, but it doesn't.
https://mirror.truenetwork.ru/CTAN/systems/win32/miktex/doc/miktex.pdf
```
initexmf --set-config-value [Core]TempDir="$env:TEMP\miktex"
```